### ISSUE TITLE
rawText returning undefined

### ISSUE THREAD

**Issue by rdgomt** [benhurott/react-native-masked-text#169]
Hello. I'm trying to use rawValue for saving in state, but always get undefined.

```
<TextInputMask
    customTextInput={TextInput}
    customTextInputProps={{
       mode: 'outlined',
          label: 'CPF',
       }}
    type={'cpf'}
    value={this.state.cpf}
    includeRawValueInChangeText={true}
    onChangeText={(maskedText, rawText) => {
        console.log(maskedText, rawText)
        this.setState({ cpf: maskedText })
    }}
/>
```

Using 1.12.0 version.

**Comment by giu-lia** [comment]
Also having this issue.

![image](https://user-images.githubusercontent.com/19209615/59983555-2a3eec80-95ef-11e9-86b7-8e968d187abf.png)

Using 1.12.3 version

![image](https://user-images.githubusercontent.com/19209615/59983559-34f98180-95ef-11e9-9f1d-7444028eaf9f.png)

**Comment by monceeef** [comment]
Same here !

**Comment by benhurott** [comment]
Hi guys!

I'm trying this sample:

```js
export default class WithRawValueInChangeText extends React.Component {
  constructor(props) {
    super(props)

    this.state = {
      value: '',
      raw: ''
    }
  }

  render() {
    return (
      <View style={container}>
        <Text>With raw value in change text</Text>
        <TextInputMask
          type={'cpf'}
          value={this.state.value}
          includeRawValueInChangeText={true}
          onChangeText={(text, rawValue) => {
            this.setState({
              value: text,
              raw: rawValue
            })
          }}
        />
        <Text>Value: {this.state.value}</Text>
        <Text>Raw: {this.state.raw}</Text>
      </View>
    )
  }
}
```

And it's working properly =/.

![image](https://user-images.githubusercontent.com/3187772/61597594-d07b1380-abe8-11e9-9bfb-d268f6ea1f64.png)

Could you provide some additional info, like:

* Are you using typescript?
* Device and OS versions

**Comment by ckristhoff** [comment]
It doesn't works with phone number or custom mask.

**Comment by sorakrisc** [comment]
> Hi guys!
> 
> I'm trying this sample:
> 
> ```js
> export default class WithRawValueInChangeText extends React.Component {
>   constructor(props) {
>     super(props)
> 
>     this.state = {
>       value: '',
>       raw: ''
>     }
>   }
> 
>   render() {
>     return (
>       <View style={container}>
>         <Text>With raw value in change text</Text>
>         <TextInputMask
>           type={'cpf'}
>           value={this.state.value}
>           includeRawValueInChangeText={true}
>           onChangeText={(text, rawValue) => {
>             this.setState({
>               value: text,
>               raw: rawValue
>             })
>           }}
>         />
>         <Text>Value: {this.state.value}</Text>
>         <Text>Raw: {this.state.raw}</Text>
>       </View>
>     )
>   }
> }
> ```
> 
> And it's working properly =/.
> 
> ![image](https://user-images.githubusercontent.com/3187772/61597594-d07b1380-abe8-11e9-9bfb-d268f6ea1f64.png)
> 
> Could you provide some additional info, like:
> 
> * Are you using typescript?
> * Device and OS versions
> 
> If you want, you can check this repo: https://github.com/benhurott/react-native-masked-text-samples

it would be great if it works on every type not only cpf

**Comment by SmirnovM91** [comment]
Do you have any ideas how to fix it?

**Comment by nandobas** [comment]
> Do you have any ideas how to fix it?

Yeah, this object is a type NativeSyntheticEvent<TextInputChangeEventData>
for get value use: value.nativeEvent.text

Or

use onChangeText, instead of onChange
